### PR TITLE
docs: Update contribution guidelines

### LINKED ISSUE #8199: docs: Outdated documentation contribution guidelines
The web version of the docs contribution guidelines here are outdated:

https://docs.meltano.com/contribute/docs#checkout-and-serve-docs-locally

They refer to installing a ruby environment, I tried the steps as listed and the `bundle install` command was unable to find the `Gemfile`, this appears to have been the approach until recently. The guidelines here in docs/README.md are more accurate:

https://github.com/meltano/meltano/blob/main/docs/README.md

### PR DESCRIPTION
**Pull request by mjsqu**
Closes https://github.com/meltano/meltano/issues/8199

Would appreciate some style tips about how to reference the version of nodejs - I tried to serve the docs using `npm start` with nodejs 12.22.12 installed (my stock Debian version), but got some errors:

```log
> docs.meltano.com@3.1.0 start
> docusaurus start

file:///home/mj2squ/repos/meltano/docs/node_modules/@docusaurus/core/bin/docusaurus.mjs:30
process.env.BABEL_ENV ??= 'development';
                       ^

SyntaxError: Unexpected token '?'
    at Loader.moduleStrategy (internal/modules/esm/translators.js:133:18)
    at async link (internal/modules/esm/module_job.js:42:21)
```

These were fixed by installing nodejs 20.8.0 from https://github.com/nodesource/distributions

### CODE REVIEW

**Comment by mjsqu** [comment]
P.S. The new docusaurus docs look really great, and serving them (once I'd updated node) is a breeze 😄

File: docs/docs/contribute/docs.md [hunk 1]
```diff
@@ -17,12 +17,11 @@ We recognize that not everything will fall into the four categories and welcome
...
 1. Fork, clone or download this project.
-1. Install ruby dependencies: `bundle install`
 1. Install node dependencies: `npm install`
-1. Build and preview: `bundle exec jekyll serve`
-1. Preview the site at [http://127.0.0.1:4000](http://127.0.0.1:4000).
+1. Run `npm start` to run a local development server
+1. View the docs at [http://localhost:3000](http://localhost:3000).
 1. Make changes to the content of the site and preview them at the link above.
 
 **Note:** Changes to `_config.yml` require you to stop the Jekyll server (`^C`) and restart it with `bundle exec jekyll serve`.
```
Comment (edgarrmondragon) [inline docs/docs/contribute/docs.md:27]: Let's get rid of this 😅

**Review by edgarrmondragon — CHANGES_REQUESTED** [review]
Hey @mjsqu! Thanks for taking the time

**Comment by mjsqu** [comment]
There are a few more references to Jekyll later on the page, would be good if someone can work out if there's a new alternative or if these sections can be removed

**Review by edgarrmondragon — APPROVED** [review]

**Comment by edgarrmondragon** [comment]
https://github.com/meltano/meltano/issues/8202

**Review by tayloramurphy — APPROVED** [review]

**Pull request merged**